schug/load/ensembl.py

```diff
@@ -109,18 +109,3 @@ def fetch_ensembl_exon_lines(
     filters = {"chromosome_name": chromosomes}
 
     return fetch_ensembl_biomart(attributes=attributes, filters=filters, build=build)
-
-
-def fetch_ensembl_exons(
```

Comment:
This function is redundant, and wasn't the one used to download the exons anyway